### PR TITLE
[IMP] l10n_vn: update chart of account

### PR DESCRIPTION
**Pull request by tong-odoo**
Impacted Version:

- 17.2 and above

This commit improve below features:

- Update chart of account based on government document
- https://vnc-os.com/wp-content/uploads/2018/09/Dual-Vie-Eng-Chart-Of-Accounts-for-Accounting-in-Vietnam-Circular-200.pdf

task-3695880

### CODE REVIEW

**Review by vin-odoo — APPROVED** [review]
Technically nothing to say, awaiting a final functional test before moving forward

File: addons/l10n_vn/data/template/account.account-vn.csv [hunk 1]
```diff
@@ -1,75 +1,81 @@
...
+"chart2134","Trademarks and brand name","2134","asset_non_current","False","Nhãn hiệu, tên thương mại"
 "chart2135","Computer software","2135","asset_non_current","False","Chương trình phần mềm"
-"chart2136","Licenses and franchises","2136","asset_non_current","False","Giấy phép và giấy phép nhượng quyền"
+"chart2136","Licenses & franchises","2136","asset_non_current","False","Giấy phép và giấy phép nhượng quyền"
 "chart2138","Other intangible fixed assets","2138","asset_non_current","False","TSCĐ vô hình khác"
 "chart2141","Depreciation of tangible fixed assets","2141","asset_non_current","False","Hao mòn TSCĐ hữu hình"
-"chart2142","Depreciation of finance lease assets","2142","asset_non_current","False","Hao mòn TSCĐ thuê tài chính"
-"chart2143","Amortization of intangible assets","2143","asset_non_current","False","Hao mòn TSCĐ vô hình"
+"chart2142","Depreciation of financial leased assets","2142","asset_non_current","False","Hao mòn TSCĐ thuê tài chính"
+"chart2143","Depreciation of intangiblen fixed assets","2143","asset_non_current","False","Hao mòn TSCĐ vô hình"
```
Comment (lordkrandel) [inline addons/l10n_vn/data/template/account.account-vn.csv:66]:
```suggestion
"chart2143","Depreciation of intangible fixed assets","2143","asset_non_current","False","Hao mòn TSCĐ vô hình"
```

File: addons/l10n_vn/data/template/account.account-vn.csv [hunk 2]
```diff
@@ -127,18 +133,18 @@
 "chart4112","Capital surplus","4112","equity","False","Thặng dư vốn cổ phần"
 "chart4113","Conversion options on convertible bonds","4113","equity","False","Quyền chọn chuyển đổi trái phiếu"
 "chart4118","Other capital","4118","equity","False","Vốn khác"
-"chart412","Differences upon asset revaluation","412","equity","False","Chênh lệch đánh giá lại tài sản"
+"chart412","Revaluation differences on asset","412","equity","False","Chênh lệch đánh giá lại tài sản"
```
Comment (lordkrandel) [inline addons/l10n_vn/data/template/account.account-vn.csv:136]: “Reevaluate.” Merriam-Webster.com Dictionary, Merriam-Webster, https://www.merriam-webster.com/dictionary/reevaluate.

File: addons/l10n_vn/data/template/account.account-vn.csv [hunk 3]
```diff
@@ -147,37 +153,38 @@
...
+"chart6411","Employees costs","6411","expense","False","Chi phí nhân viên"
 "chart6412","Materials and packing materials","6412","expense","False","Chi phí nguyên vật liệu, bao bì"
 "chart6413","Tools and instruments","6413","expense","False","Chi phí dụng cụ, đồ dùng"
-"chart6414","Fixed asset deprecation","6414","expense_depreciation","False","Chi phí khấu hao TSCĐ"
+"chart6414","Fixed asset depreciation","6414","expense_depreciation","False","Chi phí khấu hao TSCĐ"
 "chart6415","Warranty expenses","6415","expense","False","Chi phí bảo hành"
 "chart6417","Outside services","6417","expense","False","Chi phí dịch vụ mua ngoài"
 "chart6418","Other expenses","6418","expense","False","Chi phí bằng tiền khác"
-"chart6421","Staff expenses","6421","expense","False","Chi phí nhân viên"
+"chart6421","Management Employees costs","6421","expense","False","Chi phí nhân viên"
```
Comment (lordkrandel) [inline addons/l10n_vn/data/template/account.account-vn.csv:185]:
```suggestion
"chart6421","Employees management costs","6421","expense","False","Chi phí nhân viên"
```

**Comment by tong-odoo** [comment]
@robodoo r+

**Comment by tong-odoo** [comment]
@vin-odoo Hello, FC confirmed the task and can go forward. Thanks!

**Comment by vin-odoo** [comment]
@robodoo r+

**Comment by tong-odoo** [comment]
@robodoo retry

**Pull request closed without merging**